kernelci.org/content/en/blog/posts/2021/hackfests.md

```diff
@@ -0,0 +1,135 @@
+---
+date: 2021-09-20
+title: "KernelCI Hackfests"
+linkTitle: "KernelCI Hackfests"
+author: Guillaume Tucker
+description: >
+  Lessons learned from the first two hackfests
+---
+
+KernelCI hackfests span over a few days during which a number of contributors
+get together to focus on upstream Linux kernel testing.  So far, mainly kernel
+and automated test system developers have been taking part in the hackfests but
+anyone is welcome to join.  Topics mostly include extending test coverage in
+various ways: enabling new test suites as well as adding test cases to
+established frameworks such as kselftest and LTP, building additional kernel
+flavours, bringing up new particular types of hardware to be tested...
+
+There have been two hackfests to this day.  The current plan is to hold one
```

Comment:
```suggestion
There have been two hackfests to date. The current plan is to hold one
```

Reply:
Ah yes, "to this day" is for something that continues in time.  The hackfests are over now!